unlock-app/src/__tests__/reducers/errorReducer.test.js

```diff
@@ -1,48 +1,45 @@
-import React from 'react'
-import reducer from '../../reducers/errorReducer'
-import { SET_ERROR } from '../../actions/error'
+import reducer, { initialState } from '../../reducers/errorReducer'
+import { setError, resetError } from '../../actions/error'
 import { SET_PROVIDER } from '../../actions/provider'
 import { SET_NETWORK } from '../../actions/network'
 
 describe('error reducer', () => {
-  const error = <p>Something was wrong</p>
+  const action = setError({ message: 'hi', context: 'there' })
```

Comment:
Same comment as on other PR: why doing that in *this* PR? I think this is out of scope.